### PR TITLE
Fix/map clean up

### PR DESCRIPTION
**Pull request by plumdumpling**
- clean up map instance and scripts correctly
- make sure Google Maps script is added only once
- improve readability of Google Maps script parameters

### CODE REVIEW

File: library/src/google-map.ts [hunk 1]
```diff
@@ -30,9 +30,18 @@ export default class GoogleMap {
   }
 
   /**
-   * Loads the google maps script
+   * Loads the Google Maps script
    */
   private loadGoogleScript(options: GoogleMapOptions): void {
+    // Check if script tag was already added
+    if (
+      document.querySelector(
```
Comment (BiniCodes) [inline library/src/google-map.ts:38]: What about using an `id` for the script tag and getting it by id here?

File: library/src/google-map.ts [hunk 2]
```diff
@@ -30,9 +30,18 @@ export default class GoogleMap {
...
+        'script[src^="https://maps.googleapis.com/api/js"]'
+      )
+    ) {
+      return;
+    }
+
     const scriptTag = document.createElement('script');
```
Comment (BiniCodes) [inline library/src/google-map.ts:45]: [see comment above]

`scriptTag.id = 'maps-api'`

File: library/src/google-map.ts (hunk 1 above)
Comment (plumdumpling) [reply library/src/google-map.ts]: I was thinking about this too, but then there was this three headed monkey and ... you know. I'll add an id 😄

Comment (BiniCodes) [reply library/src/google-map.ts]: :D Yeah that stupid monkey problem, I'll get it! 

Happily welcomed the id 👯

**Review by BiniCodes — APPROVED** [review]

**Pull request merged**